### PR TITLE
chapter 03 (Saeid´s Homework)

### PR DESCRIPTION
**Pull request by saeidnavaser**
Hi Ahmad can you see if this pull request is correct or not...

### CODE REVIEW

File: .idea/JavaScript_course.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AhmedBHameed) [inline .idea/JavaScript_course.iml:1]: What is that file !!!

File: HOMEWORKS/Chapter_03/Saeid/Homework_01/assets/js/script.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        }
+
+        if (foot >= 232) {
+            footBall();
+            msg_03 += "This is a football.";
+        }
+        let el = document.getElementById("football");
+        el.innerHTML = msg_03;
+    },
+};
```
Comment (AhmedBHameed) [inline HOMEWORKS/Chapter_03/Saeid/Homework_01/assets/js/script.js:61]: Please do not use `getter` and `setter`. I know it is right but nothing from what we already taken!

File: HOMEWORKS/Chapter_03/Saeid/Homework_02/assets/js/script.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// We send new price value
```
Comment (AhmedBHameed) [inline HOMEWORKS/Chapter_03/Saeid/Homework_02/assets/js/script.js:116]: No accepted this amount of JS coding. Try to make your thoughts out of the box.

File: HOMEWORKS/Chapter_03/Saeid/Homework_02/index.html [hunk 4]
```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
```
Comment (AhmedBHameed) [inline HOMEWORKS/Chapter_03/Saeid/Homework_02/index.html:1]: Focusing on showing data on HTML leading to static pages. We are learning JS to make dynamic pages.

**Review by AhmedBHameed — CHANGES_REQUESTED** [review]

**Comment by saeidnavaser** [comment]
I close that pull request  and send new

**Pull request closed without merging**